### DISCUSSION TITLE
Migration down (or rollback) with custom TableName

### DISCUSSION THREAD

**Discussion by dli-eonix** [fluentmigrator/fluentmigrator#1759] [Q&A] [answered]
Hello,

I have try (without success) to migrate down or rollback a project where FM is setup.
Metadata for VersionInfo table have been changed and `TableName` is now called `_VersionInfo`

Maybe I missed the point in the documentation but I can't find how to specify a specific TableName in Console or dotnet-fm

Can you help me on this ?

Thank you,

David

**Comment by jzabroski** [comment, marked as answer]
If you are overriding behavior, the in-process runner is the recommended approach. You can use dependency injection to wire up an alternative version info Metadata.
